tests/testCmd.py

```diff
@@ -56,7 +56,7 @@ def tearDown(self):
                 for file in subdirs:
                     os.rmdir(os.path.join(dir,file))
             os.rmdir(newprod)
-                    
```

Comment:
Most of them were not intentional (one was). I have switched to Atom editor last week and it seems to have insisted on removing trailing whitespace without me asking it to do so.